build.sbt

```diff
@@ -419,18 +418,15 @@ lazy val readme = scalatex
       os.shell.call(s"git add -A", cwd = repo.getAbsolutePath)
       val nothingToCommit = "nothing to commit, working directory clean"
       try {
-        val currentUrl = s"https://github.com/scalameta/scalameta/tree/" + os.git
```

Comment:
This is an annoying side-effect of leaving line breaks in infix applications to the user, if you put a newline before `s"` it will stay that way.